src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -138,6 +139,13 @@ private static (List<Error>, TableOfContentsModel) LoadTocModel(JToken tocToken)
                 tocModelItem.TocHref = resolvedTocHref;
                 tocModelItem.Name = tocModelItem.Name ?? resolvedTopicName;
                 tocModelItem.Items = subChildren?.Items ?? tocModelItem.Items;
+
+                // validate
+                // todo: how to do required validation in strong model
```

Comment:
Any idea here? @yufeih  @live1206 ?

Reply:
If the name is not defined in input model, then we don't know the line info when reporting error.